client/classic/upe/payment-processing.js

```diff
@@ -0,0 +1,310 @@
+ */
+function submitForm( jQueryForm ) {
+	jQueryForm.removeClass( 'processing' ).trigger( 'submit' );
+}
+
+/**
+ * Creates a Stripe payment method by calling the Stripe API's createPaymentMethod with the provided elements
+ * and billing details. The billing details are obtained from various form elements on the page.
+ *
+ * @param {Object} api The API object used to call the Stripe API's createPaymentMethod method.
+ * @param {Object} elements The Stripe elements object used to create a Stripe payment method.
+ * @param {Object} jQueryForm The jQuery object for the form being submitted.
+ * @param {string} paymentMethodType The type of Stripe payment method to create.
+ * @return {Promise<Object>} A promise that resolves with the created Stripe payment method.
+ */
+function createStripePaymentMethod(
+	api,
+	elements,
+	jQueryForm,
+	paymentMethodType
+) {
+	let params = {};
+	if ( jQueryForm.attr( 'name' ) === 'checkout' ) {
+		params = {
+			billing_details: {
+				name: document.querySelector( '#billing_first_name' )
+					? (
+							document.querySelector( '#billing_first_name' )
+								?.value +
+							' ' +
+							document.querySelector( '#billing_last_name' )
+								?.value
+					  ).trim()
+					: undefined,
+				email: document.querySelector( '#billing_email' )?.value,
+				phone: document.querySelector( '#billing_phone' )?.value,
+				address: {
+					city: document.querySelector( '#billing_city' )?.value,
+					country: document.querySelector( '#billing_country' )
+						?.value,
+					line1: document.querySelector( '#billing_address_1' )
+						?.value,
+					line2: document.querySelector( '#billing_address_2' )
+						?.value,
+					postal_code: document.querySelector( '#billing_postcode' )
+						?.value,
+					state: document.querySelector( '#billing_state' )?.value,
+				},
+			},
+		};
+	}
+
+	return api
+		.getStripe( paymentMethodType )
+		.createPaymentMethod( { elements, params } );
+}
+
+/**
+ * Mounts the existing Stripe Payment Element to the DOM element.
+ * Creates the Stipe Payment Element instance if it doesn't exist and mounts it to the DOM element.
+ *
+ * @todo Make it only Split when implemented.
+ *
+ * @param {Object} api The API object.
+ * @param {string} domElement The selector of the DOM element of particular payment method to mount the UPE element to.
+ **/
+export async function mountStripePaymentElement( api, domElement ) {
+	/*
+	 * Trigger this event to ensure the tokenization-form.js init
+	 * is executed.
+	 *
+	 * This script handles the radio input interaction when toggling
+	 * between the user's saved card / entering new card details.
+	 *
+	 * Ref: https://github.com/woocommerce/woocommerce/blob/2429498/assets/js/frontend/tokenization-form.js#L109
+	 */
+	const event = new Event( 'wc-credit-card-form-init' );
+	document.body.dispatchEvent( event );
+
+	const paymentMethodType = domElement.dataset.paymentMethodType;
+	let upeElement;
+
+	// Non-split PE. To be removed.
```

Comment:
> This section can be removed when we stop supporting the legacy PE?

Exactly. More specifically, this can be removed when we implement using split PE. 

We could fully switch to using deferred intents, but we'll still need this until we split the payment elements in the frontend in separate WooCommerce gateways.